packages/blockchain-wallet-v4-frontend/src/components/Flyout/InterestUploadDocuments/UploadAndVerify.tsx

```diff
@@ -112,10 +112,18 @@ const UploadAndVerify: React.FC<Props> = (props) => {
     link.click()
   }
 
-  const fileDelete = (name: string) => {
-    setUploadAndVerified({ ...uploadAndVerified, [name]: false })
-    setUploadAndVerifiedError({ ...uploadAndVerifiedError, [name]: false })
-    setUploadedFiles({ ...uploadedFiles, [name]: null })
+  const fileDelete = (name: string, keepSecond?: boolean, secondName?: string) => {
```

Comment:
This should receive an object as parameter

Reply:
I can do it, but this is local usage inside file, maight be a bit overkill

Reply:
fixe ;)